concepticondata/references/references.bib

```diff
@@ -4385,3 +4385,21 @@ @phdthesis{Oliveira2014
   address = {Brasília},
   url     = {https://repositorio.unb.br/handle/10482/17129},
 }
+
+@article{Brown1837,
+  author      = {Brown, Nathan},
+  journal     = {Journal of the Asiatic Society of Bengal (Calcutta)},
+  number      = {2},
+  pages       = {1023-1038},
+  title       = {Comparison of Indo-Chinese languages},
```

Comment:
I think a capital L is missing for the word _Languages_ in the title.

Reply:
Not necessarily, only if you capitalize the title.

Reply:
the original has it capitalised so I'll enforce that in the  ref.